src/data-model-providers/codegen/CodegenDataModelProvider_Write.cpp

```diff
@@ -102,6 +103,10 @@ DataModel::ActionReturnStatus CodegenDataModelProvider::WriteAttribute(const Dat
     //       and tests and implementation
     //
     //       Open issue that needs fixing: https://github.com/project-chip/connectedhomeip/issues/33735
+
+    // First check things that are not even in metadata. These are read.only.
+    VerifyOrReturnValue(!IsSupportedGlobalAttributeNotInMetadata(request.path.mAttributeId), Status::UnsupportedWrite);
```

Comment:
This does not match spec, if it's actually the status that will get returned, for cases when the cluster does not exist.

So either the cluster existence check is done before we ever get here (?) or we have no test coverage for this case.

Reply:
We probably have no test coverage. I will check the ordering.